index.html

```diff
@@ -48,4 +48,178 @@ <h2 class="hex connect">LET’S CONNECT</h2>
                     <a href="#"><img src="images/socials/Vector2.svg" /></a>
                 </li>
             </ul>
-        </section>
\ No newline at end of file
+        </section>
+        <div class="works">
+            <section class="grid-items project-1">
```

Comment:
- [ ] Please make sure the `work section` is` section` tag instead of `div `tag. Using the `section` element over a generic container element like `div` can help make your code more accessible and understandable to search engines, browsers, assistive technologies, and other developers.